samples/projection-management/Program.cs

```diff
@@ -0,0 +1,312 @@
+			await managementClient.AbortAsync("countEvents_Abort");
+			// It is safe to call abort multiple times 
+			await managementClient.AbortAsync("countEvents_Abort");
+			#endregion Abort
+
+		}
+
+		private static async Task Reset(EventStoreProjectionManagementClient managementClient) {
+
+			try {
+				var js =
+					"fromAll() .when({$init:function(){return {count:0};},$any:function(s, e){s.count += 1;}}).outputState();";
+				await managementClient.CreateContinuousAsync("countEvents_Reset", js);
+			} catch (InvalidOperationException e) when (e.Message.Contains("Conflict")) {
+				// ignore 
+			}
+			
+			#region Reset
+			// Checkpoint will be written prior to resetting the projection
+			await managementClient.ResetAsync("countEvents_Reset");	
+			// It is safe to call abort multiple times 
+			await managementClient.ResetAsync("countEvents_Reset");
+			#endregion Reset
+
+		}
+
+		private static async Task CreateOneTime(EventStoreProjectionManagementClient managementClient) {
+
+			const string js =
+				"fromAll() .when({$init:function(){return {count:0};},$any:function(s, e){s.count += 1;}}).outputState();";
+			await managementClient.CreateOneTimeAsync(js);
+
+		}
+
+		private static async Task CreateContinuous(EventStoreProjectionManagementClient managementClient) {
+
+			#region CreateContinuous
+			try {
+				var js =
+					"fromAll() .when({$init:function(){return {count:0};},$any:function(s, e){s.count += 1;}}).outputState();";
+				await managementClient.CreateContinuousAsync("countEvents_Create", js);
+
+
+			} catch (InvalidOperationException e) when (e.Message.Contains("Conflict")) {
+				Console.WriteLine($"countEvents_Create already exists");
+			}
+			#endregion CreateContinuous
+
+
+		}
+
+
+
+		private static async Task Update(EventStoreProjectionManagementClient managementClient) {
+
+			#region Update
+			const string js = "fromAll().when({$init:function(){return {count:0};},$any:function(s, e){s.count += 1;}}).outputState();";
+			var name = $"countEvents_Update_{Guid.NewGuid()}";
+
+			await managementClient.CreateContinuousAsync(name, "fromAll().when()");
+			await managementClient.UpdateAsync(name, js);
+			
+			
+			try {
+				await managementClient.UpdateAsync("countEvents_Update_Not_existing", js);
+			} catch (InvalidOperationException e) when (e.Message.Contains("NotFound")) {
+				
+				Console.WriteLine("countEvents_Update_Not_existing does not exists and can not be updated");
+			}
+			#endregion Update
+		}
+
+		private static async Task ListAll(EventStoreProjectionManagementClient managementClient) {
+			#region ListAll
+			var details = managementClient.ListAllAsync();
+			await foreach (var detail in details) {
+				Console.WriteLine(
+					$@"{detail.Name}, {detail.Status}, {detail.CheckpointStatus}, {detail.Mode}, {detail.Progress}");
+			}
+			#endregion ListAll
+		}
+		
+		
+		private static async Task ListContinuous(EventStoreProjectionManagementClient managementClient) {
```

Comment:
See comment above: https://github.com/EventStore/EventStore-Client-Dotnet/pull/143/files#r686593194

Reply:
it needs to be buildable and trying to have the same name as the actual operation called, this is not visible in the doc

Reply:
correction : Listall list *all° projections , the system one as well, we need this